Allow other text in tip request.

x676f64 opened the pull request:
This PR fixes https://github.com/gibwork/gibwork-github-tips/issues/1

x676f64 committed:
Allow other text in tip request.

x676f64 committed:
Update find-first-mention.ts

x676f64 committed:
Update find-first-mention.ts

Brunakalline reviewed (approved):
https://github.com/settings/tokens

x676f64 committed:
Update find-first-mention.ts

More robust regex that allows for multiple spaces or missing spaces. Test cases here: https://regex101.com/r/jbHjKD/2

x676f64 committed:
Update find-first-mention.ts

More robust regex that allows for multiple or missing spaces in the match string. Test cases can be seen here: https://regex101.com/r/jbHjKD/3

anthonyliriano commented:
@gibworkbot tip 10 @x676f64
